### PR TITLE
Handle allowlist update and validation on the deamon side

### PR DESCRIPTION
**Pull request by bartoszWojciechO**

### CODE REVIEW

File: cli/cli_allowlist_remove_ports.go [hunk 1]
```diff
@@ -79,51 +64,17 @@ func (c *cmd) AllowlistRemovePorts(ctx *cli.Context) error {
...
-	resp, err := c.client.SetAllowlist(context.Background(), &pb.SetAllowlistRequest{
-		Allowlist: allowlist,
+	resp, err := c.client.UnsetAllowlist(context.Background(), &pb.SetAllowlistRequest{
+		Request: &pb.SetAllowlistRequest_SetAllowlistPortsRequest{
+			SetAllowlistPortsRequest: &pb.SetAllowlistPortsRequest{
+				IsUdp: isUDP,
+				IsTcp: isTCP,
+				PortRange: &pb.PortRange{
+					Start: startPort,
+					Stop:  startPort,
```
Comment (mariusSincovici) [inline cli/cli_allowlist_remove_ports.go:74]: `Stop:  startPort,` => `Stop:  endPort`,

File: cli/cli_allowlist_remove_ports.go [hunk 2]
```diff
@@ -79,51 +64,17 @@ func (c *cmd) AllowlistRemovePorts(ctx *cli.Context) error {
...
-	resp, err := c.client.SetAllowlist(context.Background(), &pb.SetAllowlistRequest{
-		Allowlist: allowlist,
+	resp, err := c.client.UnsetAllowlist(context.Background(), &pb.SetAllowlistRequest{
+		Request: &pb.SetAllowlistRequest_SetAllowlistPortsRequest{
+			SetAllowlistPortsRequest: &pb.SetAllowlistPortsRequest{
+				IsUdp: isUDP,
+				IsTcp: isTCP,
+				PortRange: &pb.PortRange{
+					StartPort: startPort,
+					EndPort:   startPort,
```
Comment (mariusSincovici) [inline cli/cli_allowlist_remove_ports.go:74]: The end port value is wrong, it needs to be `endPort` instead of `startPort`

**Review by devzbysiu — APPROVED** [review]

File: config/allowlistlist.go [hunk 3]
```diff
@@ -36,6 +36,64 @@ type Allowlist struct {
 	Subnets Subnets `json:"subnets"`
 }
 
+func (a *Allowlist) UpdateUDPPorts(ports []int64, remove bool) {
```
Comment (keliramu) [inline config/allowlistlist.go:39]: Why use have here `int64` when `uint16` is enough for values in range `1...65535`?

**Review by keliramu — APPROVED** [review]

File: config/allowlistlist.go (hunk 3 above)
Comment (bartoszWojciechO) [reply config/allowlistlist.go:39]: `int64` is used all throughout the app to represents ports, so I wanted to maintain compatibility. I'll add a task to tech debt.

**Pull request merged**